dbfit-java/core/src/main/java/dbfit/api/DbObject.java

```diff
@@ -1,12 +1,11 @@
 package dbfit.api;
 
-import dbfit.fixture.StatementExecution;
 import dbfit.util.DbParameterAccessor;
 import dbfit.util.Direction;
 
 import java.sql.SQLException;
 
 public interface DbObject {
-    public StatementExecution buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException ;
+    public DbCommand buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException;
```

Comment:
Maybe yes, it could be (as far as I see in #463 it was pretty similar). And still it's good to compare with the clean version of #541 without `DbCommand` - to see what we don't like in the case with less API changes.